lib/styleguide.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const _ = require('lodash'),
+  clayUtils = require('clayutils'),
+  files = require('nymag-fs'),
+  path = require('path');
+
+/**
+ * Looks for CSS files associated with a component's variation
+ *
+ * @param {String} styleguide - name of the styleguide that is set in the sites config
+ * @returns {Array} variations
+ */
+function findVariations(styleguide) {
```

Comment:
Think this might be better named something like `findVariationFiles` to distinguish it from `getVariations`